splinterd/tests/admin/registry.rs

```diff
@@ -0,0 +1,451 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Integration tests for the registry REST API endpoints
+
+use crate::framework::network::Network;
```

Comment:
The order of use statements here should be:

```
use std::collections::HashMap;

use splinter::registry::client::RegistryNode;
use splinterd::node::RestApiVariant;

use crate::framework::network::Network;
```

Basically less-to-more specific, in groups (std, crates, crate::, super, children modules).

Reply:
Noted and fixed